src/hooks/Timeline.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { VideoContext } from '../shared/VideoContext'
+
+const baseClass = 'timeline'
+
+export const Timeline = props => {
+  const { duration, currentTime } = React.useContext(VideoContext)
+  const percentComplete = Math.floor((currentTime / duration) * 100)
+  const onMouseMove = evt => {
+    const mouseWidth = evt.pageX
```

Comment:
`mouseWidth` seems ... weird here. That sounds like a constant that is the width in pixels of the cursor. Is this more like `mousePosition` or `clickPositionX` or something? 🤷‍♂